### ISSUE TITLE
Proposal for main contributors

### ISSUE THREAD

**Issue by juliohm** [JuliaPlots/Plots.jl#1156]
Could someone already involved with the codebase perhaps host a webinar or record a video going over the machinery in Plots.jl in an attempt to educate potential contributors?

I am particularly interested in tackling all the issues tagged with GR because that is my favorite backend. However, I currently don't have the time to learn how Plots.jl does things internally, the layers of abstraction, etc. If I had a "push" from someone that could explain the details, I could keep going by myself.

**Comment by apalugniok** [comment]
There is a rather old video from tbreloff. Giving an overview of Plots.jl internals, have you seen that one?

**Comment by mkborregaard** [comment]
It's a good idea, if it leads to people like you having an easier time contributing. One thing to appreciate at length is this call graph, showing what happens when you call `histogram(x)`. https://drive.google.com/open?id=0BwWjA0Ez4iLhMGVkTVZMXzBUQkk  Notice half the work is done by `display`, which gets automatically called when you return a Plot object to the console.

**Comment by mkborregaard** [comment]
Sorry, wrong link - here's the right one: https://drive.google.com/open?id=0BwWjA0Ez4iLhdERRRVlHekZiejA

**Comment by mkborregaard** [comment]
Here's the good thing: `plot` calls are largely identical, so once you know this call graph well you'll know it all.

**Comment by juliohm** [comment]
Thank you @apalugniok, where is the link?

Thank you @mkborregaard, we definitely need someone going through the internals in a video or something, this call graph is insane :flushed:

**Comment by juliohm** [comment]
Yes, it would be great to have something more specific like fixing an issue with one backend for example.

**Comment by mkborregaard** [comment]
There's also this http://docs.juliaplots.org/latest/contributing/
The answer to that last question is to go to the code for that backend and fix it :joy:

**Comment by juliohm** [comment]
I think what I will do is start watching the PRs. If you could explain the changes in the PR description at least, that would be already a good addition to the documentation.

It would be great to have a CONTRIBUTING file explaining that all PRs could contain a careful explanation of the changes, what the issue was and how it was solved.

I will close this one and follow the repository.
